pkgs/desktops/lxde/core/lxappearance/default.nix

```diff
@@ -1,26 +1,42 @@
-{ stdenv, fetchurl, intltool, pkgconfig, libX11, gtk2, withGtk3 ? false, gtk3 }:
+{ stdenv
+, fetchurl
+, intltool
+, pkg-config
+, libX11
+, gtk2
+, gtk3
+, withGtk3 ? true
+}:
 
 stdenv.mkDerivation rec {
   name = "lxappearance-0.6.3";
 
-  src = fetchurl{
+  src = fetchurl {
     url = "mirror://sourceforge/project/lxde/LXAppearance/${name}.tar.xz";
     sha256 = "0f4bjaamfxxdr9civvy55pa6vv9dx1hjs522gjbbgx7yp1cdh8kj";
   };
 
-  nativeBuildInputs = [ pkgconfig intltool ];
+  nativeBuildInputs = [
+    pkg-config
+    intltool
+  ];
 
-  buildInputs = [ libX11 (if withGtk3 then gtk3 else gtk2) ];
+  buildInputs = [
+    libX11
+    (if withGtk3 then gtk3 else gtk2)
+  ];
 
-  patches = [ ./lxappearance-0.6.3-xdg.system.data.dirs.patch ];
+  patches = [
+    ./lxappearance-0.6.3-xdg.system.data.dirs.patch
+  ];
 
   configureFlags = stdenv.lib.optional withGtk3 "--enable-gtk3";
 
-  meta = {
-    description = "A lightweight program for configuring the theme and fonts of gtk applications";
+  meta = with stdenv.lib; {
+    description = "Lightweight program for configuring the theme and fonts of gtk applications";
     homepage = "https://lxde.org/";
-    maintainers = [ stdenv.lib.maintainers.hinton ];
-    platforms = stdenv.lib.platforms.linux;
-    license = stdenv.lib.licenses.gpl2;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
+    maintainers = with  maintainers; [ hinton romildo ];
```

Comment:
Does `nixpkgs-fmt` really insert this double space here?

Reply:
Fixed. There were some minor manual editions that was not mentioned in the commit message.